sdk/monitor/azure-monitor-opentelemetry-exporter/samples/logs/sample_log_aad_managed.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""
+An example to show an application using Opentelemetry logging sdk. Logging calls to the standard Python
+logging library are tracked and telemetry is exported to application insights with the AzureMonitorLogExporter.
+"""
+import os
+import logging
+
+from azure.identity import ManagedIdentityCredential
+from opentelemetry._logs import (
+    get_logger_provider,
+    set_logger_provider,
+)
+from opentelemetry.sdk._logs import (
+    LoggerProvider,
+    LoggingHandler,
+)
+from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+
+from azure.monitor.opentelemetry.exporter import AzureMonitorLogExporter
+
+set_logger_provider(LoggerProvider())
+credential = ManagedIdentityCredential(client_id="<client_id>")
```

Comment:
I don't think we need a separate aad sample for each signal. Maybe just create a folder called authenticate and have one example for manage and secret?

Reply:
Done